api/requirements.in

```diff
@@ -1,8 +1,14 @@
+# Define all requirements, only pin when necessary (and if so, add a comment explaining why).
+# See # https://git.datapunt.amsterdam.nl/Datapunt/python-best-practices/blob/master/dependency_management/
+
+django<3
```

Comment:
Hier moet nog een comment bij waarom je < 3 pakt.

Reply:
Volgens mij is dit wat je nodig hebt:

`# not updating to 3.0. See https://github.com/jpadilla/django-rest-framework-xml/issues/45`